### PR TITLE
SqlPeerDB fix Name shadowed: peers

### PR DESCRIPTION
**Pull request by llybin**

### CODE REVIEW

File: src/main/kotlin/brs/db/sql/SqlPeerDb.kt [hunk 1]
```diff
@@ -14,9 +14,9 @@ internal class SqlPeerDb(private val dp: DependencyProvider) : PeerDb {
 
     override fun updatePeers(peers: List<String>) {
         dp.db.useDslContext { ctx ->
-            val peers = ctx.selectFrom(PEER).fetch(PEER.ADDRESS)
-            ctx.deleteFrom(PEER).where(PEER.ADDRESS.notIn(peers))
-            ctx.batch(peers.mapNotNull { if (peers.contains(it)) null else ctx.insertInto(PEER).set(PEER.ADDRESS, it) })
+            val peerEntities = ctx.selectFrom(PEER).fetch(PEER.ADDRESS)
```
Comment (harry1453) [inline src/main/kotlin/brs/db/sql/SqlPeerDb.kt]: Could you perhaps use `dbPeers` as a name instead? It makes it clearer what the value actually is

File: src/main/kotlin/brs/db/sql/SqlPeerDb.kt [hunk 2]
```diff
@@ -14,9 +14,9 @@ internal class SqlPeerDb(private val dp: DependencyProvider) : PeerDb {
...
+            ctx.deleteFrom(PEER).where(PEER.ADDRESS.notIn(peerEntities))
```
Comment (harry1453) [inline src/main/kotlin/brs/db/sql/SqlPeerDb.kt]: This should be peers, not peerEntities

File: src/main/kotlin/brs/db/sql/SqlPeerDb.kt [hunk 3]
```diff
@@ -14,9 +14,9 @@ internal class SqlPeerDb(private val dp: DependencyProvider) : PeerDb {
...
+            ctx.batch(peerEntities.mapNotNull { if (peerEntities.contains(it)) null else ctx.insertInto(PEER).set(PEER.ADDRESS, it) })
```
Comment (harry1453) [inline src/main/kotlin/brs/db/sql/SqlPeerDb.kt]: This should be

```kotlin
ctx.batch(peers.mapNotNull { if (peerEntities.contains(it)) null else ctx.insertInto(PEER).set(PEER.ADDRESS, it) })
```

Basically first peerEntities should be peers

**Comment by harry1453** [comment]
Why close?

**Comment by llybin** [comment]
I have not a great English. May be entities is not a best naming. You know it's a hardest things what we solve everyday, what name to give for a variable xD but DB for me is not clear, what is dbpeers?  For me maybe it's database of peers... :) Peers is okay, but busy :)

**Pull request closed without merging**